app/vmalert/group.go

```diff
@@ -49,6 +50,8 @@ func (g *Group) Restore(ctx context.Context, q datasource.Querier, lookback time
 // updateWith updates existing group with
 // passed group object.
 func (g *Group) updateWith(newGroup Group) {
+	g.Interval = newGroup.Interval
```

Comment:
hm, I forgot about this... So hot reload of rules will require to update ticker in `group.start` if interval has changed. I'm not sure if dynamic ticker update will work - this need to be tested. Alternatively, group may be stopped and started again after interval change.

Reply:
@hagen1778 OK, I Will do it in next week!

Reply:
Unfortunately, I don't see fix for this issue in the PR. You can verify it by writing the test which will check Eval intervals after applied update.
As I pointed above, you might need to restart group to make it work properly.

Reply:
Fixed